gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -152,42 +152,26 @@ def compute_hard_filters(cov_threshold: int) -> hl.Table:
 
 
 def compute_sex() -> hl.Table:
-    # Compute sex chrom poloidy
-    ht = impute_sex_ploidy(
-        get_gnomad_v3_mt(remove_hard_filtered_samples=False),
-        excluded_calling_intervals=telomeres_and_centromeres.ht()
+    mt = get_gnomad_v3_mt(
```

Comment:
this is imported with `from gnomad_qc.v3.resources import *`, right? basic question, is there a reason why this is better here than importing each function individually?

Reply:
Yes it is, I think I would prefer to import each individually, but will need to go back and change all that in another PR at somepoint